src/libsystemd-network/dhcp-server-internal.h

```diff
@@ -40,6 +40,9 @@ typedef struct DHCPLease {
 
 struct sd_dhcp_server {
         struct in_addr relay_target;
+        char *agent_circuit_id;
+        char *agent_remote_id;
```

Comment:
Not a matter, but I prefer to move these variable at somewhere below, maybe end of the struct definition, as these are not important in most usecases.